storage/engine/mvcc.go

```diff
@@ -1591,21 +1591,25 @@ func MVCCComputeStats(iter Iterator, nowNanos int64) (MVCCStats, error) {
 	ms := MVCCStats{LastUpdateNanos: nowNanos}
 	first := false
 	meta := &MVCCMetadata{}
+	scratch := make([]byte, 0, 128)
```

Comment:
Is there value to allocating an array on the stack and building a slice out of that?